Makefile

```diff
@@ -60,7 +60,7 @@ devnet-clean:
 	rm -rf ./.devnet
 	cd ./ops-bedrock && docker-compose down
 	docker image ls | grep ops-bedrock_ | cut -d ' ' -f 1 | xargs docker rmi
```

Comment:
```
docker image ls 'ops-bedrock*' --format='{{.Repository}}' | xargs docker rmi
```

Sorry I should have posted both at the same time, was working on getting the second 1 liner